ci/common/common.sh

```diff
@@ -153,7 +153,7 @@ commit_subtree() {(
 
 with_token() {(
   set +x
-  set +o pipefail
+  set +e
```

Comment:
My intention here is to let `sed` act on any output resulting from `xargs` in the pipeline on line 167. I didn't test this though.